src/components/MultiStepForm/index.tsx

```diff
@@ -40,6 +40,33 @@ export interface Values {
 }
 
 export default function MultiStepForm() {
+	const router = useRouter();
+
+	async function onSubmit(values: Values) {
+		const response = await fetch("/handle-request", {
+			method: "POST",
+			headers: {
+				"Content-Type": "application/json",
+			},
+			body: formatValues(values),
+		});
+		console.log(response);
+		if (!response.ok) {
+			throw new Error(response.statusText);
+		}
+		const data = await response.json();
+
+		let redirectEndpoint = "/acolhimento-andamento";
+
+		for (let key in data) {
+			if (data[key] && data[key] != "duplicated") {
+				redirectEndpoint = "/cadastro-finalizado";
+			}
+		}
+
+		router.push(redirectEndpoint);
```

Comment:
nao entendi essa parte

Reply:
@vivianedias é até algo que esqueci de comentar, eu fique pensando no caso de ela pedir dois acolhimentos e um for solicitação repetida e outro uma recebida. Aí coloquei que se pelo menos um pedido não for repetida iria para tela de cadastro-finalizado e aí pelo email ela veria o que aconteceu com cada um.